resources/assets/js/app/views/topics/topicCreate.js

```diff
@@ -0,0 +1,52 @@
+var Backbone = require('backbone');
+var Marionette = require('backbone.marionette');
+var TopicModel = require('../../Models/TopicModel');
+
+module.exports = Marionette.ItemView.extend({
+    template: 'topicCreateNew',
+    model: new TopicModel(),
+
+    ui: {
+        name: '[name="name"]',
+        description: '[name="description"]',
+        createForm: '.topic-form'
+    },
+
+    initialize: function () {
+        this.bindUIElements();
+    },
+
+    modelEvents: {
+        'invalid': function (model, errors, options) {
+            $('.errors').empty();
+            for (var error in errors) {
+                $('[name="' + error + '"]').siblings('.errors').html(errors[error]);
+            }
+        }
+    },
+
+    events: {
+        'change @ui.name': function () {
+            this.model.set({name: this.ui.name.val()})
+        },
+        'change @ui.description': function () {
+            this.model.set({description: this.ui.description.val()})
+        },
```

Comment:
you no need to describe all input items, use one selector that will contains all input values, and set value in model depends on input attr

It's not a bug, just suggestion, you no need to change this logic